src/web/components/SmallHeadline.tsx

```diff
@@ -61,6 +61,20 @@ const slashStyles = css`
     }
 `;
 
+const linkStyles = css`
+    position: relative;
+
+    color: inherit;
+    :active :hover :visited {
```

Comment:
This surprised me, the lack of `,` between?

Reply:
It surprised you because it was bogus. I missed the missing commas because all we need here is `color: inherit;`

I've removed the unused css.